### PR TITLE
feat: zksync subgraph

### PR DESCRIPTION
**Pull request by KolevDarko**
Bumped libraries and created subgraph config file for zkSyncEra.
ZK sync Era is only supported by The Graph's Subgraph studio, so our automated flow could not be used.
I updated the README with the instructions to update the zksync subgraph from [here](https://thegraph.com/studio/subgraph/request-payment-zksyncera/)

### CODE REVIEW

File: cli/graph-nodes.ts [hunk 1]
```diff
@@ -40,8 +40,16 @@ export const graphNodeInfoByNetwork: GraphNodeInfoByNetwork = {
     index: "https://thegraph.coredao.org/graphql",
     ipfs: "https://thegraph.coredao.org/ipfs/",
   },
+  // Zksync era deployment was done via subgraph studio
+  zksyncera: {
+    queryBase: "https://api.studio.thegraph.com/query/35843/request-payment-zksyncera/version/latest",
+    deploy: "",
+    index: "",
+    ipfs: ""
+  }
```
Comment (KolevDarko) [inline cli/graph-nodes.ts:49]: I could not find these urls for this subgraph in the [Studio page](https://thegraph.com/studio/subgraph/request-payment-zksyncera/)
@benjlevesque is that ok?

Comment (benjlevesque) [reply cli/graph-nodes.ts:49]: you should only reference chains that aren't supported by the studio. You can remove zksyncera from this list

File: README.md [hunk 2]
```diff
@@ -137,6 +137,19 @@ For decentralized network, use:
...
+
+Step 1: Authenticate the graph-cli with subgraph studio. Get the studio token (here)[https://thegraph.com/studio/subgraph/request-payment-zksyncera/]
+```
+graph auth --studio <studio token>
+```
+Step 2: Deploy to subgraph studio
+```
+graph deploy --studio request-payment-zksyncera subgraph.zksyncera.yaml
+```
+
```
Comment (benjlevesque) [inline README.md:152]: isn't this the same process as the decentralized network? cc @alexandre-abrioux

**Review by benjlevesque — APPROVED** [review]

**Comment by MantisClone** [comment]
What is the public URL for the Payments Subgraph on zkSync Era? Does it have one?

**Comment by MantisClone** [comment]
Oh never mind. I found it.
https://github.com/RequestNetwork/requestNetwork/blob/f1f2f91a4a90e9aef854aa23162acc944dedc779/packages/payment-detection/src/thegraph/client.ts#L20

**Pull request merged**